src/apps/checklist-button/checklist-material-button.test.js

```diff
@@ -26,9 +26,10 @@ describe("Checklist material button", () => {
     cy.contains("Tilføj til din huskeliste").click();
     cy.contains("Tilføjet");
     cy.contains("Fjern fra din huskeliste").click();
-    cy.contains("Fjernet").click();
+    cy.contains("Fjernet");
+    cy.contains("Tilføj til din huskeliste");
   });
-  it("Can remove a material from the checklist and readd it", () => {
+  it("Can remove an initial material from the checklist", () => {
```

Comment:
Why do you want to delete the readding part?

It is actually part of the test case and the previous test case also says "Can add a material to the checklist **and remove it**"